hardware/raspberrypi/spi/README.md

```diff
@@ -136,6 +151,23 @@ When asking for say 24 MHz, the actual speed will be 15.6 MHz.
 
 Forum post: [SPI has more speeds](https://www.raspberrypi.org/forums/viewtopic.php?f=44&t=43442&p=347073)
 
+### bcm2835 library
+
+This is a C library for Raspberry Pi (RPi). It provides access to GPIO and other IO functions on the Broadcom BCM 2835 chip. Accesses the hardware registers directly.
+
+http://www.airspayce.com/mikem/bcm2835/
+
+### Use spidev from C
+
+There's a loopback test program in the Linux documentation that can be used as a starting point. See the [Troubleshooting](#troubleshooting) section. Uses the Linux `spidev` driver to access the bus.
+
```

Comment:
This is likely to need more detail, but OK for the moment just to get things up to date.